### PR TITLE
[ja] Translate sidecar-container.md into Japanese

### PR DESCRIPTION
**Pull request by yuto-kimura-g**
### Description

Translate 
`content/en/docs/reference/glossary/sidecar-container.md` 
into Japanese
`content/ja/docs/reference/glossary/sidecar-container.md`

### Issue

Closes: #48851 

### robot

/area localization
/language ja

### CODE REVIEW

File: content/ja/docs/reference/glossary/sidecar-container.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: サイドカーコンテナ
+id: sidecar-container
+date: 2024-11-28
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/sidecar-container.md:4]:
```suggestion
date: 2018-04-12
```

nit: "date" はオリジナルに準拠してください

https://github.com/kubernetes/website/blob/d63058f01edd345c03d0a6a3d1bc4c184a18525f/content/en/docs/reference/glossary/sidecar-container.md?plain=1#L4

**Review by Okabe-Junya — COMMENTED** [review]
Thank you for your contribution!!

/approve

Deploy preview: https://deploy-preview-48878--kubernetes-io-main-staging.netlify.app/ja/docs/reference/glossary/?all=true#term-sidecar-container

**Comment by yuto-kimura-g** [comment]
@Okabe-Junya 
Thank you for your review.
I have fixed the date, please confirm.

**Review by Okabe-Junya — COMMENTED** [review]
LGTM
we need another look from b1gb4by (or other ja-reviewer) to move forward

**Comment by yuto-kimura-g** [comment]
I see.
@b1gb4by san, could you please do a review?

**Comment by sftim** [comment]
Nice work IMO.

**Review by b1gb4by — APPROVED** [review]
/lgtm

**Comment by Okabe-Junya** [comment]
JFYI > @b1gb4by 

[細かい仕様] GitHub上のレビューで "Comment" ではなく "Approve" してしまうと、Prow に `/approve` として解釈されてしまうので、`/lgtm` する際は "Comment" した方が安全そうです


`/approve` せずに `approved` ラベルが付与される例: https://github.com/kubernetes/website/pull/48886#pullrequestreview-2483795312

**Comment by b1gb4by** [comment]
@Okabe-Junya 
なるほど、承知しました🙇

**Pull request merged**